### ISSUE TITLE
b-modal deep css not working after upgrade [rc releases]

### ISSUE THREAD

**Issue by yaniv51** [bootstrap-vue/bootstrap-vue#4723]
### Describe the bug
After upgrading bootstrap-vue from 2.0.0-rc.18/2.0.0-rc.19 to higher version, all of our b-modal components does not received the deep CSS
### Steps to reproduce the bug

Because of PortalVue  usage since version 2.0.0-rc.20, the DOM location of b-modal is not at the component section.
While using scoped deep css, we get css like:
<style scoped>
>>> .size-table-modal {
    	background-color:rgba(255,0,0,0.52941);
}
</style>
The output is:
[data-v-xxxx] .size-table-modal {
    	background-color:rgba(255,0,0,0.52941);
}

but the data-v-xxxx is no more conected to the b-modal since it is in the body end.

### Expected behavior

scoped deep css selector should effect modal elements

### Versions

**Libraries:**

- BootstrapVue: > 2.0.0-rc.19

### Additional context

We found that when working with global style - it is working as expected. Did I misses something and the only solution is to work with global style or there is a simple solution?
As I mentioned, in versions < 2.0.0-rc.20 the css is working fine.

Thanks

**Comment by tmorehouse** [comment]
b-modal should inherit the correct scoped data attribute (we have special code to handle getting hte scope ID).

Are you using `v2.0.0+` ? or are you still using `2.0.0-rc##` releases?  The scoping was added in 2.0.0-stable

**Comment by tmorehouse** [comment]
See https://bootstrap-vue.js.org/docs/misc/changelog#v200

> **Notable improvements:**
> [ ... ]
> - Modals, tooltips, popovers, and toasts now work with scoped style classes (requires the use of vue-loader's `/deep/`, `::v-deep` or `>>>` deep selectors for targeting inner elements, just like with any other component).
>
> [ ... ]

**Comment by yaniv51** [comment]
Hi @tmorehouse ,
I forgot to update you that I've figure out why it is happen.

In our case, we used to put the b-modal in some cases, in other bootstrap vue component.
For example
<b-form-group>
whatever
<b-modal>
</b-modal>
</b-form-group>

When you specified that the b-modal should inherit scoped data attribute - I realized that the parent of our b-modal is not correct.
It was working on rc-18 but not on new version.
